.github/libs/GithubUtils.js

```diff
@@ -223,23 +225,25 @@ class GithubUtils {
 
                 // PR list
                 if (!_.isEmpty(sortedPRList)) {
-                    issueBody += '\r\n**This release contains changes from the following pull requests:**\r\n';
+                    issueBody += '\r\n**This release contains changes from the following pull requests:**';
                     _.each(sortedPRList, (URL) => {
-                        issueBody += _.contains(verifiedPRList, URL) ? '- [x]' : '- [ ]';
-                        issueBody += ` ${URL}\r\n`;
+                        issueBody += `\r\n\r\n${URL}`;
+                        issueBody += _.contains(verifiedPRList, URL) ? '\r\n  - [x] QA' : '\r\n  - [ ] QA';
+                        issueBody += _.contains(accessabilityPRList, URL) ? '\r\n  - [x] Accessibility' : '\r\n  - [ ] Accessibility';
                     });
                 }
 
                 // Deploy blockers
                 if (!_.isEmpty(deployBlockers)) {
-                    issueBody += '\r\n**Deploy Blockers:**\r\n';
+                    issueBody += '\r\n\r\n\r\n**Deploy Blockers:**';
                     _.each(sortedDeployBlockers, (URL) => {
-                        issueBody += _.contains(resolvedDeployBlockers, URL) ? '- [x]' : '- [ ]';
-                        issueBody += ` ${URL}\r\n`;
+                        issueBody += `\r\n\r\n${URL}`;
+                        issueBody += _.contains(resolvedDeployBlockers, URL) ? '\r\n  - [x] QA' : '\r\n  - [ ] QA';
+                        issueBody += _.contains(accessabilityPRList, URL) ? '\r\n  - [x] Accessibility' : '\r\n  - [ ] Accessibility';
```

Comment:
If we're not creating such a thing as Accessibility-based DeployBlockers, then why add this to this section? Also, it's important to note that, in NewDot, _all deploy blockers are issues_. We don't allow pull requests to be deploy blockers in-and-of-themselves because then we can't auto-assign them internally using the `Engineering` label.

This is all to say I don't think we need to make any changes to the deploy blocker section for now.

Reply:
Word, I'll change it there. I was not aware that all deploy blockers are issues (not pull requests) that's just how it was in the code so I kept it /shrug